cypress/page/dresses-list.page.ts

```diff
@@ -0,0 +1,27 @@
+class DressesListPage {
+
+    private dressItem: string;
+    private dressName: string;
+  
+    constructor(){
+      this.dressItem = ".product-container"
+      this.dressName = `${this.dressItem} .product-name`
+    }
+  
+    getDressProducts(){
+      return cy.get(this.dressItem)
+    }
+  
+    validateItemsNumber(itemsNumber: number){
+      cy.get(this.dressItem).should("have.length", itemsNumber)
+    }
+  
+    validateItemsNames(names: string[]){
+      cy.get(this.dressName).each((item, index) => {
+        cy.wrap(item).should("contain.text", names[index])
+      })
+    }
+  
+  }
+  
+  export {DressesListPage}
```

Comment:
```suggestion
export {DressesListPage}
```
and EOF.

Reply:
In general check the identation of this class, seems like a mix of 4 and 2 spaces.

Reply:
resolved